cli/main.rs

```diff
@@ -518,6 +519,59 @@ async fn doc_command(
   }
 }
 
+async fn doctest_command(
+  flags: Flags,
+  include: Option<Vec<String>>,
+  fail_fast: bool,
+  quiet: bool,
+  allow_none: bool,
+  filter: Option<String>,
+) -> Result<(), ErrBox> {
+  let global_state = GlobalState::new(flags.clone())?;
+  let cwd = std::env::current_dir().expect("No current directory");
```

Comment:
style: for expect strings I try to always put the expectation that should hold true rather than an error message indicating what you think went wrong.
```suggestion
  let cwd = std::env::current_dir().expect("expected: process has a current working directory");
```